html/webappapis/user-prompts/newline-normalization.manual.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Newline normalization in simple dialogs</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#concept-dialog-newline-normalization">
+
+<p>The dialogs should all contain text looking like:</p>
+
+<pre>First line
+Second line</pre>
+
+<p>or:</p>
```

Comment:
Don't you want to require one or the other?

Reply:
Yes, this is "inclusive or"?

Reply:
I mean for each test, say if 1 or 2 newlines are expected

Reply:
I guess we could, but it's not possible for a test that was meant to output "line" to instead say "paragraph", so I'm not sure what value it would add.